tensorboard/webapp/app_routing/effects/app_routing_effects.ts

```diff
@@ -462,6 +462,9 @@ export class AppRoutingEffects {
                     ? options
                     : {
                         ...options,
+                        namespaceUpdate: {
+                          option: NamespaceUpdateOption.UNCHANGED,
+                        },
                         replaceState: true,
                         resetNamespacedState: false,
```

Comment:
If you don't mind doing it while you're here, I think we can also delete resetNamespacedState here. Seems like it is forgotten cleanup from a previous refactor. As far as I can tell it is not actually part of NavigationOptions and it is not used anywhere.